src/package.json

```diff
@@ -65,12 +65,11 @@
     "@types/react-test-renderer": "^18.0.0",
     "babel-jest": "^29.6.4",
     "eslint": "^8.47.0",
-    "jest": "^29.6.4",
+    "jest": "^29.2.1",
```

Comment:
why to downgrade?

Reply:
Good catch, I'll update to the one recommended here:
https://react-native-community.github.io/upgrade-helper/?from=0.73.8&to=0.75.2